{
    "title": "pytorch-lightning is vulnerable to Deserialization of Untrusted Data",
    "score": 7.8,
    "severity": "HIGH",
    "cwe_id": "CWE-502",
    "cvss_vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
    "cpe_match": [
        "cpe:2.3:a:pytorchlightning:pytorch_lightning:*:*:*:*:*:python:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": []
}
